etc/avocado/avocado.conf

```diff
@@ -1,10 +1,10 @@
 [datadir.paths]
 # Avocado data dir (holds tests and test auxiliary data, such as ISO files).
-base_dir = /usr/share/avocado
+base_dir = /var/lib/avocado
 # You may override the specific test directory with test_dir
 test_dir = /usr/share/avocado/tests
 # You may override the specific test auxiliary data directory with data_dir
-data_dir = /usr/share/avocado/data
+data_dir = /var/lib/avocado/data
```

Comment:
Well isn't the `/var/lib` purpose to store variable data? If so then I think we should omit the `data` dir at the end as it's already implied by the `/var/lib`.